src/base/__tests__/Context.ts

```diff
@@ -1200,12 +1208,108 @@ describe('Context class', () => {
         }
       );
 
-      result = await context.issuedClaims();
+      result = (await context.issuedClaims()) as ResultSet<ClaimData>;
```

Comment:
These assertions shouldn't be necessary once the function is returning a single type